hack/util.sh

```diff
@@ -63,7 +63,7 @@ capz::util::should_build_ccm() {
 
 # all test regions must support AvailabilityZones
 capz::util::get_random_region() {
-    local REGIONS=("canadacentral" "eastus" "eastus2" "northeurope" "uksouth" "westeurope" "westus2" "westus3")
+    local REGIONS=("australiaeast" "canadacentral" "eastus" "eastus2" "northcentralus" "northeurope" "uksouth" "westeurope" "westus2")
```

Comment:
I've seen a few failures lately in eastus2 that look like the westus3 ones. Does availability/quota look ok there from the new sub?

Reply:
Standard BS Family CPUs: 0 / 350
Standard DSv3 Family CPUs: 0 / 200

The subscription shows zero usage currently for anything in East US 2. Those are the two SKU families used generally by CAPZ testing, and neither of them is flagged as constrained in the Azure portal. (Although in practice, that doesn't mean they aren't actually in limited supply.)

Do you know which SKU type specifically?

Reply:
Actually this may not be VM SKU related. Here's one recent failure with logs: https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/capz-conformance-master/1815146168741531648.

And another that ran in eastus2 without logs: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cloud-provider-azure/6638/pull-cloud-provider-azure-e2e-ccm-vmss-capz-eks-test/1815297774736904192

Reply:
And I can see that the conformance tests have passed in eastus2 since that failure. False alarm? 🤞 

https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/capz-conformance-master/1815373674300051456

Reply:
Now I see a couple of capacity errors in [westeurope](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/5000/pull-cluster-api-provider-azure-e2e-aks/1815443064311779328) and [eastus2](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/5001/pull-cluster-api-provider-azure-e2e-aks-v1beta1/1815443318062977024) for Standard_D2s_v3.

Reply:
We have no SKUs running in eastus2 and only 16 running in westeurope at the moment. None of the regions have any substantial utilization currently (well under quota).

So as you said, must be just a capacity error. At some point if it's frequent, we should remove the less reliable regions from the list, as we did for the previous e2e sub.

Reply:
We also have the intended quota in `francecentral` and `germanywestcentral` now, if we need to replace unreliable locations.